### PR TITLE
Remove FAB rotation and change duration

### PR DESCRIPTION
**Pull request by BrayanDSO**
A few changes that I personally thought would be good on top of #12280 

# Before

[device-2023-06-10-152148.webm](https://github.com/ankidroid/Anki-Android/assets/69634269/a656f255-93fb-41b8-be5e-07f8b6e629d4)

# After

[device-2023-06-10-151626.webm](https://github.com/ankidroid/Anki-Android/assets/69634269/16d7a507-626b-407b-b9d5-252c9dd141d7)

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/DeckPickerFloatingActionMenu.kt [hunk 1]
```diff
@@ -80,17 +80,15 @@ class DeckPickerFloatingActionMenu(
                     mFabMain.backgroundTintList = ColorStateList.valueOf(fabNormalColor)
                 }
                 // Rotates FAB to 90 degrees
```
Comment (lukstbit) [inline AnkiDroid/src/main/java/com/ichi2/anki/DeckPickerFloatingActionMenu.kt:82]: Maybe we could delete this comment as well

File: AnkiDroid/src/main/java/com/ichi2/anki/DeckPickerFloatingActionMenu.kt [hunk 2]
```diff
@@ -144,16 +142,15 @@ class DeckPickerFloatingActionMenu(
                 // Close with animation
                 mFabMain.animate().apply {
                     // Rotates FAB to 180 degrees
```
Comment (lukstbit) [inline AnkiDroid/src/main/java/com/ichi2/anki/DeckPickerFloatingActionMenu.kt:144]: This one as well.

**Review by lukstbit — APPROVED** [review]
Ok.

**Review by mikehardy — APPROVED** [review]
Yeah I think removing rotation makes more sense now that the add icon is different
Duration change seems fine - certainly can't think why not, so --> works for me :-), thanks!

**Pull request merged**